### PR TITLE
[Addon] add the mysql-exporter addon

### PR DESCRIPTION
**Pull request by barnettZQG**
Signed-off-by: barnettZQG <barnett.zqg@gmail.com>

### Checklist

I have:

- [x] Title of the PR starts with type (e.g. `[Addon]` , `[example]` or `[Doc]`).
- [ ] Updated/Added any relevant [documentation](https://kubevela.io/docs/reference/addons/overview) and [examples](../examples).
- [ ] New addon should be put in [experimental](../experimental/addons).
- [x] Update addon should modify the `version` in `metadata.yaml` to generate a new version.
- [x] Any changes about [verified](../addons) addons should be tested with CI [script](../test/e2e-test/hack/addon-vela-test.sh).

### CODE REVIEW

File: addons/mysql-exporter/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+description: Prometheus exporter for MySQL server metrics.
+icon: https://www.mysql.com/common/logos/powered-by-mysql-167x86.png
+url: https://github.com/prometheus/mysqld_exporter
+
+tags:
+  - Observability
+
+deployTo:
+  controlPlane: true
+  runtimeCluster: false
```
Comment (wangyikewxgm) [inline addons/mysql-exporter/metadata.yaml:13]: deploy to runtime cluster?

File: addons/mysql-exporter/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: mysql-exporter
+version: 0.0.1
+appVersion: v1.14.0
```
Comment (wangyikewxgm) [inline addons/mysql-exporter/metadata.yaml:3]: What's appVersion ?

File: addons/mysql-exporter/template.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: mysql-operator
```
Comment (wangyikewxgm) [inline addons/mysql-exporter/template.yaml:4]: mysql-exporter?

Comment (barnettZQG) [reply addons/mysql-exporter/template.yaml:4]: Actually, the name here is unuseful.

File: addons/mysql-exporter/metadata.yaml (hunk 2 above)
Comment (barnettZQG) [reply addons/mysql-exporter/metadata.yaml:3]: We should have an appVersion field same as the helm chart. The version field only represents the addon.

File: addons/mysql-exporter/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+
+# Mysql Exporter
+
+Prometheus exporter for MySQL server metrics.
+
+Supported versions:
+
+MySQL >= 5.6.
+MariaDB >= 10.3
+
```
Comment (wangyikewxgm) [inline addons/mysql-exporter/README.md:10]: Please give a example application tell user how to use this addon.

Comment (barnettZQG) [reply addons/mysql-exporter/README.md:10]: Already added.

**Review by wangyikewxgm — APPROVED** [review]

**Pull request merged**